EastCoastGreenwayAlliance/ecg-map
# new signup form

Refer to https://github.com/GreenInfo-Network/east-coast-greenway/issues/196

- new EveryAction form
- forwards to map.greenway.org
- and sets the "Do not display this message in the future" flag so it won't ask again

Testing:
- open 2 shell windows
- in one, cd to `ecg-map`
- Check out the `newsignupform-196` branch
- in the other cd to `ecg-map/client`
- in the first one (root folder), run `npm run dev`
- in the second one (client), run `npm start`
